templates/api/knowledge-panels/contribution/data_quality_tags.tt.json

```diff
@@ -0,0 +1,68 @@
+{
+    "level": "info",
+    "topics": [
+        "contribution"
+    ],
+    [%# only expand errors and info %]
+    [% IF panel.tags_type == "data_quality_errors" %]
+        "expand_for": "large",
+    [% ELSIF panel.tags_type == "data_quality_info" %]
+        "expand_for": "large",
+    [% ELSE %]
+        "expanded": false,
+    [% END %]
+    [% IF panel.tags_type == "data_quality_errors" %]
+        "evaluation": "bad",
+        [% icon_name = "data-error" %]
+    [% ELSIF panel.tags_type == "data_quality_warnings" %]
+        "evaluation": "average",
+        [% icon_name = "data-warning" %]
+    [% ELSE %]
+        "evaluation": "good",
+        [% icon_name = "data-info" %]
+    [% END %]
+    "title_element": {
+        "title": "[% lang(panel.tags_type _ '_panel_title') %]",
+        "subtitle": "[% lang(panel.tags_type _ '_panel_subtitle') %]",
+        "icon_url": "[% static_subdomain %]/images/icons/dist/[% icon_name %].svg",
+        "icon_color_from_evaluation": true,
+    },
+    "elements": [
+        [% 
+            # we need description_key variable because we can't use ':' in key directly
+            description_key = 'description:en'
+            action_name = "action_$fix_action"
```

Comment:
I think $fix_action is only defined below in the for loop, shouldn't this be inside the for loop?